### PR TITLE
Create component Button

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Create component Button 

image

### CODE REVIEW

File: examples/practice-typescript/src/components/Button/index.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
+@import '../../styles/bases/variable.scss';
```
Comment (thanhnguyen-agilityio) [inline examples/practice-typescript/src/components/Button/index.scss]: Should this belong to general file? We cannot duplicate import it every component.

File: examples/practice-typescript/src/components/Button/index.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import './index.scss';
+
+interface ButtonProps {
+  text: string;
+  style: 'btn__action' | 'btn__add';
```
Comment (thanhnguyen-agilityio) [inline examples/practice-typescript/src/components/Button/index.tsx]: This is not style, it is a class naming for style. Pls change other name like `styleClass` or `className`

File: examples/practice-typescript/src/components/Button/index.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+.btn {
+  letter-spacing: 0;
+  color: $color-text-secondary;
+  border: 0.067rem;
+  border-radius: 1.15rem;
+
+  &__add {
```
Comment (thanhnguyen-agilityio) [inline examples/practice-typescript/src/components/Button/index.scss]: This is scss syntax. As discuss we use css only for this practice. So, you could use CSS nested https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_nesting/Using_CSS_nesting

```css
.btn {
  letter-spacing: 0;
 /**/

  .btn__add {}
}
```

File: examples/practice-typescript/src/components/Button/index.scss (hunk 1 above)
Comment (thanhtrantruong47) [reply examples/practice-typescript/src/components/Button/index.scss]: Fixed

File: examples/practice-typescript/src/components/Button/index.tsx (hunk 2 above)
Comment (thanhtrantruong47) [reply examples/practice-typescript/src/components/Button/index.tsx]: Fixed

**Review by thanhnguyen-agilityio — APPROVED** [review]

**Pull request merged**